d3a_api_client/setups/test_create_aggregator.py

```diff
@@ -53,60 +60,40 @@ def on_finish(self, finish_info):
         self.is_finished = True
 
 
-import os
-simulation_id = os.environ["API_CLIENT_SIMULATION_ID"]
-domain_name = os.environ["API_CLIENT_DOMAIN_NAME"]
-websocket_domain_name = os.environ["API_CLIENT_WEBSOCKET_DOMAIN_NAME"]
+simulation_id, domain_name, websocket_domain_name = get_simulation_info()
 
 
 aggr = TestAggregator(
-    simulation_id=simulation_id,
-    domain_name=domain_name,
     aggregator_name="test_aggr",
-    websockets_domain_name=websocket_domain_name
 )
 
 device_args = {
-    "simulation_id": simulation_id,
-    "domain_name": domain_name,
-    "websockets_domain_name": websocket_domain_name,
     "autoregister": False,
     "start_websocket": False
 }
 
 load1_uuid = get_area_uuid_from_area_name_and_collaboration_id(
-    device_args["simulation_id"], "Load", device_args["domain_name"])
+    simulation_id, "Load", domain_name)
 device_args["device_id"] = load1_uuid
 
 
 load1 = RestDeviceClient(
     **device_args
 )
 
-
-load2_uuid = get_area_uuid_from_area_name_and_collaboration_id(
-    device_args["simulation_id"], "Load 2", device_args["domain_name"])
-device_args["device_id"] = load2_uuid
-
-load2 = RestDeviceClient(
-    **device_args
-)
-
-pv1_uuid = get_area_uuid_from_area_name_and_collaboration_id(
-    device_args["simulation_id"], "PV", device_args["domain_name"])
+pv1_uuid = get_area_uuid_from_area_name_and_collaboration_id(simulation_id, "PV", domain_name)
 device_args["device_id"] = pv1_uuid
```

Comment:
This step will be obsolete once D3ASIM-3209 is implemented, right?

Reply:
As far as I see not really as its done from the device owner to get uuid of its device which is being used to `select_aggregator` whereas 3209 is for an aggregator to know devices it controls.

Reply:
Exactly, this is the aggregator-script here. If the aggregator would know the uuids of the devices it controls, this step would be not needed.

Reply:
I see now. You mean for the direct connection to the device we still need it. 
Yes, this starts the discussion again, when we stop maintaining the direct connections. Will move this discussion to the D3ASIM-3365 ticket and marking this here as resolved as it is not related.

Reply:
> Exactly, this is the aggregator-script here. If the aggregator would know the uuids of the devices it controls, this step would be not needed.

Before that device owner needs to select an aggregator too which is ATM done by the same script.

>  You mean for the direct connection to the device we still need it.

Thats a separate matter, if device owner should directly trade on behalf of their device or always select an aggregator. What I meant was selection of aggregator is done by device owner and ATM its only possible via RestDeviceClient's method `select_aggregator `, and to instantiate `RestDeviceClient ` for an specific device, one needs uuid for respective device.